rustfmt.toml

```diff
@@ -3,20 +3,19 @@ array_width = 80
 attr_fn_like_width = 80
 chain_width = 80
 comment_width = 100
-#condense_wildcard_suffixes = true
-#enum_discrim_align_threshold = 40
+condense_wildcard_suffixes = true
 fn_call_width = 80
-#fn_single_line = true
-#format_code_in_doc_comments = true
-#format_macro_matchers = true
-#format_strings = true
-#group_imports = "StdExternalCrate"
-#hex_literal_case = "Lower"
-#imports_granularity = "Crate"
+format_code_in_doc_comments = true
+format_macro_matchers = true
+group_imports = "StdExternalCrate"
```

Comment:
We probably should enable `reorder_modules = true` and `reorder_imports = true`, otherwise this setting and `imports_granularity` are only partially addressing formatting of imports.

Probably a good idea to enable `wrap_comments = true` and `reorder_impl_items = true` - but maybe all of these are defaults? I have them in the compiler's `rustfmt.toml`, so if they are implied I guess that would be good to know.

Reply:
Added `reorder_modules = true` and `reorder_imports = true` - though, I didn't change anything. I guess these are implied by some other settings.

`wrap_comments = true` was already there.

Tried out setting `reorder_impl_items = true`, but I think it re-orders things in a slightly un-intuitive way - so, reverted this change.

Reply:
Ah, good to know - thanks for testing it out!